pkg/services/ssosettings/api/api.go

```diff
@@ -76,17 +76,32 @@ func (api *Api) RegisterAPIEndpoints() {
 	})
 }
 
+// swagger:route GET /v1/sso-settings sso_settings listAllProvidersSettings
+//
+// # List all SSO Settings entries
+//
+// You need to have a permission with action `settings:read` with scope `settings:auth.<provider>:*`.
+//
+// Responses:
+// 200: okResponse
```

Comment:
I recently learned that we can define custom response types for 200 OK. See the latest version in main for the GET endpoint. There is a `getSSOSettingsResponse` response there instead of `okResponse`. Can you please do the same for the list endpoint?